src/porepy/numerics/fv/test_coupler_mpfa.py

```diff
@@ -0,0 +1,133 @@
+import numpy as np
+import time
+import scipy.sparse as sps
+import scipy.spatial as spat
+from scipy.sparse.linalg import spsolve
+import matplotlib.pyplot as plt
+
+from porepy.grids import structured, simplex, point_grid
+from porepy.grids.grid import FaceTag
+from porepy.params import bc, second_order_tensor
+
+
+# from viz.plot_grid import plot_grid, save_img
+from porepy.viz.exporter import export_vtk
+
+from porepy.grids.coarsening import *
+from porepy.grids import grid_bucket, remove_grids
+from porepy.fracs import meshing  # split_grid
+
+from porepy.numerics.mixed_dim import coupler
+import copy
+from porepy.numerics.fv import mpfa, tpfa_coupling, tpfa
+from porepy.numerics.mixed_dim import condensation
+
+#------------------------------------------------------------------------------#
+
+
+def add_data(gb):
+    """
+    Define the permeability, apertures, boundary conditions
+    """
+    gb.add_node_props(['k', 'f', 'bc', 'bc_val', 'a'])
+    for g, d in gb:
+        kxx = np.ones(g.num_cells)
+
+        if all(g.cell_centers[0, :] < 0.0001):
+            perm = second_order_tensor.SecondOrderTensor(3,
+                                                         kxx / 100, kxx, kxx)
+        else:
+            perm = second_order_tensor.SecondOrderTensor(3,
+                                                         kxx * np.power(100, g.dim < 3))
+
+        d['k'] = perm
```

Comment:
I am not happy with such non-descriptive names, but we should set standards for this on a more general level.